### DISCUSSION TITLE
Disable entire UI?

### DISCUSSION THREAD

**Discussion by Lynnesbian** [slint-ui/slint#2310] [Q&A] [answered]
Is there a way to disable every widget in the `Window` at once?

Most widgets have an `enabled` property, so it would be possible to disable them all individually, but I was wondering if there was a more efficient way of doing this.

**Comment by ogoffart** [comment, marked as answer]
This is currently not supported, but we plan to add that in the future.
There were not yet an open issue about it, so i opened #2338 for it.

**Reply by Lynnesbian** [reply]
Nice to know, thanks!
